### ISSUE TITLE
Sheeps lack color when sheared

### ISSUE THREAD

**Issue by Hirachosan** [BTW-Community/BTW-Public#56]
Just saw a video where sheared sheep had their respective color as leftover specks. Though very minor, it'd be cool to rectify this.

**Comment by ammoore00** [comment]
I don't think this is a bug per-se. But definitely would be a nice thing, especially with how hard it is to get different sheep colors in BTW. Will look into it
